network/onchain-discovery/src/client.rs

```diff
@@ -337,7 +338,8 @@ where
                                 addrs.push(addr);
                             }
                         }
-
+// TODO:  This is the UpdateAddress that I cannot conver to UpdateConfiguration because the eligible peers are not available.
```

Comment:
@davidiw 

onchain-discovery is the current blocker on consolidating the interface as it does not appear to have access to the trusted/eligible peer set.